site/networking.xml

```diff
@@ -0,0 +1,696 @@
+
+<pre class="code">
+[
+  {rabbit, [
+    {tcp_listeners, [{"192.168.1.99", 5672}]}
+  ]}
+].
+</pre>
+        </p>
+        <p>
+          Alternatively, if a single stack setup is desired, the interface can be
+          configured using the <code>RABBITMQ_NODE_IP</code> environment variable.
+          See our <a href="/configure.html">Configuration guide</a> for detalis.
+        </p>
+      </doc:subsection>
+
+      <doc:subsection name="single-stack-ipv6">
+        <doc:heading>Listening on IPv6 Interfaces Only</doc:heading>
+        <p>
+          In this example RabbitMQ will listen on an IPv6 interface only:
+
+<pre class="code">
+[
+  {rabbit, [
+    {tcp_listeners, [{"fe80::2acf:e9ff:fe17:f97b", 5672}]}
+  ]}
+].
+</pre>
+        </p>
+        <p>
+          Alternatively, if a single stack setup is desired, the interface can be
+          configured using the <code>RABBITMQ_NODE_IP</code> environment variable.
+          See our <a href="/configure.html">Configuration guide</a> for detalis.
+        </p>
+      </doc:subsection>
+    </doc:section>
+
+    <xi:include href="install-selinux-ports.xml.inc" />
+
+    <doc:section name="tls">
+      <p>
+        It is possible to encrypt connections using TLS with RabbitMQ. Authentication
+        using peer certificates is also possible. Please refer to the <a href="/ssl.html">TLS/SSL guide</a>
+        for more information.
+      </p>
+    </doc:section>
+
+    <doc:section name="tuning-for-throughput">
+      <doc:heading>Tuning for Throughput</doc:heading>
+
+      <doc:subsection name="tuning-for-throughput-intro">
+        <p>
+          Tuning for throughput is a common goal. Improvements can be achieved by
+
+          <ul>
+            <li>Increasing TCP buffer sizes</li>
+            <li>Ensuring Nagle's algorithm is disabled</li>
+            <li>Enabling optional TCP features and extensions</li>
+          </ul>
+
+          For the latter two, see the OS-level tuning section below.
+
+          Note that tuning for throughput will involve trade-offs. For example, increasing TCP buffer
+          sizes will increase the amount of RAM used by every connection, which can be a significant
+          total server RAM use increase.
+        </p>
+      </doc:subsection>
+
+      <doc:subsection name="tuning-for-throughput-tcp-buffers">
+        <doc:heading>TCP Buffer Size</doc:heading>
+        <p>
+          This is one of the key tunable parameters. Every TCP connection has buffers
+          allocated for it. Generally speaking, the larger these buffers are, the more RAM
+          is used per connection and better the throughput. On Linux, the OS will automatically
+          tune TCP buffer size by default, typically settling on a value between 80 and 120 KB.
+          For maximum throughput, it is possible to increase buffer size using the <code>rabbit.tcp_listen_options</code>.
+        </p>
+        <p>
+          The following example sets TCP buffers to 192K:
```

Comment:
The unit is missing. It should be "192 KiB" if we follow the SI units. I'm fine with "192 KB" otherwise (with a space between the number and the unit).